packages/replayer/src/replay.utils.ts

```diff
@@ -114,15 +116,31 @@ export const createReplayPage: (options: {
   return page;
 };
 
-const installVirtualEventLoop = (page: Page) =>
-  page.evaluateOnNewDocument(`
-    const installVirtualEventLoop = window.__meticulous?.replayFunctions?.installVirtualEventLoop;
-    if (installVirtualEventLoop) {
-      installVirtualEventLoop();
-    } else {
-      console.error("Could not install virtual event loop since window.__meticulous.replayFunctions.installVirtualEventLoop was null or undefined");
-    }
-`);
+const installVirtualEventLoop = (page: Page, sessionStartTime: DateTime) => {
+  const opts: InstallVirtualEventLoopOpts = {
+    sessionStartTime: sessionStartTime.toMillis(),
+  };
+  return page.evaluateOnNewDocument(`
+      const installVirtualEventLoop = window.__meticulous?.replayFunctions?.installVirtualEventLoop;
+      if (installVirtualEventLoop) {
+        installVirtualEventLoop(${JSON.stringify(opts)});
+      } else {
+        console.error("Could not install virtual event loop since window.__meticulous.replayFunctions.installVirtualEventLoop was null or undefined");
+      }
+  `);
+};
+
+export const getSessionStartTime = (sessionData: SessionData): DateTime =>
```

Comment:
Because this is exported from the package want to give it a name that doesn't assume it's calculated from rrweb data, but instead just uses whatever is our current canonical way of calculating the session start time, as expected by patchDate's and installVirtualEventLoop's sessionStartTime param